CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
 this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
The ## [Unreleased] section has been added, but the corresponding comparison link at the bottom of the changelog is missing.
Could you also add the following line to ensure proper version diff tracking?

```
[Unreleased]: https://github.com/aicers/review-web/compare/0.27.0...main
```